analysis.go

```diff
@@ -174,12 +147,13 @@ func CheckImagePullBackOff(project string, stdErr io.Writer) (Result, error) {
 // supplied project and if any have replicas set to zero this will be reflected
 // in the returned Result data. Any errors are written to the supplied stdErr
 // writer.
-func CheckDeployConfigsReplicasNotZero(project string, stdErr io.Writer) (Result, error) {
+func CheckDeployConfigsReplicasNotZero(project, basepath string) (Result, error) {
 	result := Result{Status: 0, StatusMessage: "this issue was not detected", CheckName: "check deployconfig replicas not 0"}
 	deploymentConfigs := DeploymentConfigs{}
-	err := getResourceStruct(project, "dc", &deploymentConfigs)
+	err := getDumpedResource([]string{basepath, "projects", project, "definitions", "deploymentconfigs.json"}, &deploymentConfigs)
 	if err != nil {
-		stdErr.Write([]byte(err.Error()))
+		result.Status = 2
```

Comment:
good question, I just picked a non-zero number to indicate a problem... this could probably do with being defined somewhere though, and maybe more useful codes that could be searched in documentation or something?